### DISCUSSION TITLE
Background color for active window

### DISCUSSION THREAD

**Discussion by jenstroeger** [kovidgoyal/kitty#6286] [Q&A] [answered]
Hello,

I’m exploring Kitty (and perhaps to replace my [i3wm](https://i3wm.org/)) and was wondering: is it possible to change the background color/alpha of the active window inside of a tab?

Thanks!

**Comment by kovidgoyal** [comment]
Search https://sw.kovidgoyal.net/kitty/conf/ for active

**Reply by jenstroeger** [reply]
Well that’s why I ask: I found only settings for the in/active window border color and text alpha, and the in/active tab background. But nothing regarding the background for the in/active _window_ inside of a tab.

**Comment by kovidgoyal** [comment]
the background of windows is drawn by the programs running inside the
terminal.

**Reply by jenstroeger** [reply, marked as answer]
Fair point, makes sense. So the answer to my question is “No”.

With `i3` every “tile” is its own window (OS Window in Kitty terms) so it’s possible to change them individually.
